packages/SketchMorph2-Core.package/M2Backend.class/instance/saveSettingsDuring..st

```diff
@@ -1,10 +1,11 @@
 undo
 saveSettingsDuring: aBlock
-	| preColor preFillingGeometric preTool |
+	| preBrushSize preColor preFillingGeometric preTool |
+	preBrushSize := self brushSize.
 	preColor := self currentColor.
 	preFillingGeometric := self isFillingGeometricForm.
 	preTool := self currentTool.
 	
 	aBlock value.
-	self restoreColor: preColor isFilling: preFillingGeometric tool: preTool.
+	self restoreBrushSize: preBrushSize color: preColor isFilling: preFillingGeometric tool: preTool.
```

Comment:
A cascade could be used here.

Reply:
You mean something like
```smalltalk
self restoreBrushSize: preBrushSize
    color: preColor
    isFilling: preFillingGeometric
    tool: preTool.
```
? I'm not sure this makes it more readable.

Reply:
Also keep in mind the limit for characters per line (I believe it was 80?)

Reply:
Ok, I'll change it.

Reply:
> 
> 
> You mean something like
> 
> ```smalltalk
> self restoreBrushSize: preBrushSize
>     color: preColor
>     isFilling: preFillingGeometric
>     tool: preTool.
> ```
> 
> ? I'm not sure this makes it more readable.

No I meant something like that
```smalltalk
self
    restoreBrushSize: preBrushSize color: preColor isFilling: preFillingGeometric tool: preTool;
    reloadCurrentPen
```

Reply:
Oh, ok. This looks reasonable.